### ISSUE TITLE
Missing ResponseMetadata

### ISSUE THREAD

**Issue by youngbob** [vemel/mypy_boto3_builder#33]
**Describe the bug**
Every response contains a ResponseMetadata key in addition to the response-specific data. This is not added to the response types, leading to mypy errors when attempting to access it.

**To Reproduce**
1. Install `boto3-stubs['lambda']`
2.  Run mypy on the following code sample

```python
import boto3
import json

client = boto3.client('lambda')
response = client.invoke(
    FunctionName='arn:aws:lambda:fake',
    InvocationType='Event',
    Payload=json.dumps({'message': 'hi there'}).encode('utf-8'),
)
print(response['ResponseMetadata']['RequestId'])
```
3. Actual mypy output

```
test.py:10: error: TypedDict "InvocationResponseTypeDef" has no key 'ResponseMetadata'
Found 1 error in 1 file (checked 1 source file)
```

**Expected mypy-output**

```
Success: no issues found in 1 source file
```

**Comment by youngbob** [comment]
I'm no longer seeing this after updating to the latest 1.14.19

**Comment by youngbob** [comment]
Nevermind, this is failing in a clean environment. I'm not sure what's changed here.

**Comment by vemel** [comment]
Thank you for the report. Looks like it is really missing from TypedDicts. WIll take a look.

**Comment by vemel** [comment]
Looks like I am also missing `Error`. I did not find a way to check if `ResponseMetadata` exists in response and in some cases it does not. So I will make both `ResponseMetadata` and `Error` optional keys.

**Comment by vemel** [comment]
Finally, this has been addressed. Thank you for the report and let me know if it works for you.

**Comment by youngbob** [comment]
I've removed the code that needed this at some point, but my original example here works now. Thanks!

**Comment by tuukkamustonen** [comment]
@vemel I'm on `1.14.63.0` and it doesn't seem to work:

![image](https://user-images.githubusercontent.com/94327/110295144-70252980-7ff9-11eb-9d31-ed26d32a4a80.png)

Types are missing `ResponseMetadata`:

![image](https://user-images.githubusercontent.com/94327/110295196-80d59f80-7ff9-11eb-8e61-89d33d775be3.png)

Any ideas?

**Comment by tuukkamustonen** [comment]
@vemel Hmm, similarly, I don't see `NextToken` anywhere...?
